apps/system/js/app_chrome.js

```diff
@@ -827,9 +827,19 @@
 
     if (this.isSearch()) {
       var dataset = this.app.config;
-      data.options.push({ id: 'search', text: dataset.searchName });
+      data.options.push({
+        id: 'search',
+        text: {
+          raw: dataset.searchName
+        }
+      });
     } else {
-      data.options.push({ id: 'origin', text: this.title.textContent });
+      data.options.push({
+        id: 'origin',
+        text: {
+          raw: this.title.textContent
+        }
+      });
     }
 
     ModalDialog.selectOne(data, selected);
```

Comment:
Didn't you refactor this method to take an additional non-optional `title` parameter?